traditionalVocalBeaming.qml

```diff
@@ -22,12 +22,19 @@ import MuseScore 3.0
 
 MuseScore {
     version:  "1.0"
+    id: traditionalVocalBeaming
     description: "This plugin creates traditional vocal beams, i.e. beams only on melismas."
     menuPath: "Plugins.Traditional Vocal Beaming"
 
+    Component.onCompleted : {
+        if (mscoreMajorVersion >= 4) {
+            traditionalVocalBeaming.title = "Traditional Vocal Beaming";
+        }
+    }
+
     onRun: {
 	if (typeof curScore === 'undefined')
-	    Qt.quit();
+	    quit();
```

Comment:
`Qt.quit()` should never be used in MU3 and can't in MU4. `quit()` works in MU4 and does nothing (but emitting a warning in the Plugin Creator) in MU3

Reply:
Might also change ot to `return`